webpack/assets/javascripts/react_app/components/Bookmarks/Bookmarks.stories.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { storiesOf } from '@theforeman/stories';
 import BookmarkForm from './components/BookmarkForm';
-import BookmarkModal from './components/SearchModal';
+import BookmarkModal from './components/SearchModal/SearchModal';
```

Comment:
Can you add a button to open the `BookmarkModal` in the story?

Reply:
@MariaAga Done, thanks